README.md

```diff
@@ -1,142 +1,82 @@
-Før man starter å kode er det derfor viktig å kjøre `bash get-started.sh` fra Root. Da vil alle de tre overnevnte bli 
+Før man starter å kode er det derfor viktig å kjøre `bash get-started.sh` fra Root. Da vil alle de tre overnevnte bli
 installert.
 
-Monorepo for ny saksbehandlingsløsning for ytelser til etterlatte
-
-### AzureAD secrets på lokal maskin
-
-Tokens/secrets som gjør det mulig å gå mot dev-gcp fra lokal maskin.\
-Kan hentes ved å kjøre følgende kommando:
-```
-./get-secret.sh <APP_NAME>
-```
-
-Hvis du får feilmelding `Not logged in or lacking rights. Ensure you're connected to naisdevice and gcp.`
-og du er sikker på du er koblet både naisdevice og gcp så kan det være at du må sette default namespace på context:
-
-```
-kubectl config set-context --current --namespace=etterlatte
-```
-
-
-**NB:** Ved kjøring av en eller flere backend(s) lokalt kan det være lurt å installere
-[EnvFile](https://plugins.jetbrains.com/plugin/7861-envfile) i IntelliJ. Uten denne må du manuelt kopiere
-fra .env-filen til appen sine miljøvariabler.
-
-## Maven
-Vi bruker personal access token til autentisering for å hente intern pakker til prosjektet.
-Du må derfor sette `export GITHUB_TOKEN='DITT_TOKEN'` som miljøvariabel.
-Dette tokenet må autoriseres mot navikt.
-
-## Apper
-
-[etterlatte-behandling](apps/etterlatte-behandling) \
-Tjeneste som holder styr på behandlinger.
-
-[etterlatte-beregning](apps/etterlatte-beregning) \
-// TODO
+# Lokal utvikling
 
-[etterlatte-brev-api](apps/etterlatte-brev-api) \
-Ktor og Rapid app for å håndtere generering av brev, brevmaler og sende videre til distribusjon.
+## Hvordan kjøre en eller flere apper
 
-[etterlatte-fordeler](apps/etterlatte-fordeler) \
-Fordeler aktuelle saker inn til behandling i ny applikasjon.
+Det enkleste er å kjøre opp frontend og valgfri backend med docker compose som starter alt man trenger for at APIet skal
+fungere.
 
-[etterlatte-grunnlag](apps/etterlatte-grunnlag) \
-Tjeneste som holder styr på behandlinger.
+1. **Hent secrets**
 
-[etterlatte-gyldig-soeknad](apps/etterlatte-gyldig-soeknad) \
-// TODO
+   Kjør scriptet `get-secret.sh` med appnavn.
+    ```shell
+    ./get-secret.sh <app_name>
+    ```
 
-[etterlatte-hendelser-pdl](apps/etterlatte-hendelser-pdl) \
-Lytter på hendelser fra Livet er en strøm av hendelser
+   Eksempel for henting av secrets til frontend:
+    ```shell
+    ./get-secret.sh etterlatte-saksbehandling-ui
+    ```
 
-[etterlatte-oppdater-behandling](apps/etterlatte-oppdater-behandling) \
-Oppdater behandling er en Rapid-app, som brukes til å lese meldinger relevant for behandlings-appen og sende HTTP-kall til behandlingsappen for å gjøre forskjellige handlinger.
+2. **Kjør docker compose**
+    ```shell
+    docker compose up -d
+    ```
+   NB: Hvis du vil kjøre frontend mot lokal backend må du overskrive variabel for `_API`-url (se oversikt i `config.ts`)
```

Comment:
Mener du her lokal frontend mot frackend i gcp?

Reply:
Har oppdatert og lenket til eget punkt for dette.